### PR TITLE
improve example for `stdlib`'s `parseInt` function

### PR DESCRIPTION
**Pull request by hamirmahal**

### CODE REVIEW

File: docs/src/frontend/index.md [hunk 1]
```diff
@@ -1126,12 +1126,19 @@ Parses a signed `{!rsh} Int` into a JavaScript number.
 Example:
 
 ```js
-load: /examples/programmer-art/src/index.mjs
-md5: 0779cd5bfb7d45de6a7f46b9cd6c7164
-range: 394 - 402
+import { loadStdlib } from "@reach-sh/stdlib";
```
Comment (jeapostrophe) [inline docs/src/frontend/index.md:1129]: I think you should add this as a "test-case" to `/examples/js-tests` and then cite the lines

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: examples/js-tests/parseInt/index.mjs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { loadStdlib } from "@reach-sh/stdlib";
```
Comment (jeapostrophe) [inline examples/js-tests/parseInt/index.mjs:1]: You should not make a new file. We want the tests to actually run and `js-tests` already runs

File: docs/src/frontend/index.md (hunk 1 above)
Comment (hamirmahal) [reply docs/src/frontend/index.md]: This is what `frontend/#js_parseInt` looks like after [062ef7c](https://github.com/reach-sh/reach-lang/pull/1115/commits/062ef7c27079f984589ac0cfc4616119f45e1972).

image

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs/src/frontend/index.md (hunk 1 above)
Comment (hamirmahal) [reply docs/src/frontend/index.md]: This is what `frontend/#js_parseInt` looks like after [f16706b](https://github.com/reach-sh/reach-lang/pull/1115/commits/f16706becb3514660854cfe52cb417d072cdc271).

image

**Comment by jeapostrophe** [comment]
Merge after CI

**Pull request merged**